scipy/signal/filter_design.py

```diff
@@ -260,26 +260,29 @@ def freqz(b, a=1, worN=None, whole=False, plot=None):
     Given the M-order numerator `b` and N-order denominator `a` of a digital
     filter, compute its frequency response::
 
-                 jw               -jw               -jwM
-        jw    B(e  )  b[0] + b[1]e    + .... + b[M]e
-     H(e  ) = ---- = -----------------------------------
-                 jw               -jw               -jwN
-              A(e  )  a[0] + a[1]e    + .... + a[N]e
+                 jw                 -jw              -jwM
+        jw    B(e  )    b[0] + b[1]e    + ... + b[M]e
+     H(e  ) = ------ = -----------------------------------
+                 jw                 -jw              -jwN
+              A(e  )    a[0] + a[1]e    + ... + a[N]e
 
     Parameters
     ----------
     b : array_like
-        Numerator of a linear filter. Must be 1D.
+        Numerator of a linear filter.
     a : array_like
-        Denominator of a linear filter. Must be 1D.
+        Denominator of a linear filter.
```

Comment:
The docstring will need to be updated to describe the broadcasting behavior of `b, a, worN`, i.e. if I have multidimensional versions of all of these how do I get them to broadcast properly?

Reply:
(and if it's complicated, adding a small example might be justified)

Reply:
See if the updated version looks sufficient.